internal/legacylibrarian/legacyautomation/trigger_test.go

```diff
@@ -125,7 +125,7 @@ func TestRunCommandWithClient(t *testing.T) {
 					Id:   "publish-release-trigger-id",
 				},
 			},
-			ghPRs:           []*legacygithub.PullRequest{{HTMLURL: github.Ptr("https://github.com/googleapis/librarian/pull/1")}},
+			ghPRs:           []*legacygithub.PullRequest{{HTMLURL: github.Ptr("https://cloud.google.com/go/librarian/pull/1")}},
```

Comment:
Should this one be updated? It seems like the urls should still be the github URL
```suggestion
			ghPRs:           []*legacygithub.PullRequest{{HTMLURL: github.Ptr("https://github.com/googleapis/go/librarian/pull/1")}},
```

Reply:
technically this won't break because we run automation based on `legacy-librarian` branch. But this should not be changed and I will revert. Thanks for noticing!